src/Workspaces/Core/Portable/Options/EditorConfig/IEditorConfigStorageLocation2.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Options
+{
+    internal interface IEditorConfigStorageLocation2 : IEditorConfigStorageLocation
+    {
+        /// <summary>
+        /// Gets the editorconfig string representation "key = value" for this storage location.
```

Comment:
nit: i would remove the "key=value" part, since it won't apply if we do this for NamingConfig.  Don't want to imply this is only for that specific output form.